docs/src/components/Markdown.js

```diff
@@ -65,7 +66,7 @@ const formatComponentName = (listitem) => {
   return listitem;
 };
 
-export default function Markdown({ text, type }: Props): Node {
+export default function Markdown({ color, text, type }: Props): Node {
```

Comment:
just a thought.. would `textColor` be more explicit? I wasn't sure if color was background etc

Reply:
mmm good call, will update